storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -810,6 +820,10 @@ bool Archived_Redo_Log_Reader::seek_logfile(lsn_t lsn) {
   auto file_size = my_tell(file, MYF(MY_FAE));
 
   if (file_size < pos) {
+#ifdef UNIV_DEBUG
```

Comment:
yes make sense,  and we can use that flag in the test case.

Reply:
I have added an `DBUG_PRINT(archive..` in the code. In case we see failure . We can pass `--debug=d,archive` to run.sh to see more details about failure.